tamr_unify_client/client.py

```diff
@@ -64,14 +64,19 @@ def __init__(
         self.port = port
         self.base_path = base_path
         self.session = session or requests.Session()
-
```

Comment:
Extraneous whitespace change; please remove.

Reply:
@CaspianA1 : this is a case where we want this newline (for readability purposes), so removing it is not only extraneous but incorrect. When @nbateshaus says "please remove", he means "please remove this change" (i.e. don't remove the newline), not "please remove the whitespace".